src/fann_train.c

```diff
@@ -40,6 +40,10 @@ fann_type fann_activation_derived(unsigned int activation_function, fann_type st
     case FANN_LINEAR_PIECE:
     case FANN_LINEAR_PIECE_SYMMETRIC:
       return (fann_type)fann_linear_derive(steepness, value);
+    case FANN_LINEAR_PIECE_LEAKY:
+      return (fann_type)((value < 0) ? steepness * 0.01 : steepness);
```

Comment:
It would be probably good idea to have negative slope tunable. Although it doesn't need to be in this PR as we would use 0.01 as default anyway.

Reply:
I agree this would be a good idea, I can open an issue for it and other people can contribute the enhancement.